contributors/devel/contributor-experience/contributor-github-cheat-sheet.en.md

```diff
@@ -0,0 +1,286 @@
+# GitHub Cheat Sheet
+
+# Contributor GitHub Cheat Sheet
+
+This document contains tips, tricks, and common best practices used within the
+Kubernetes project. It is a "TL;DR" or quick reference of useful information to
+make your GitHub contribution experience better.
+
```

Comment:
- In general, I like the format, and way the information is presented. It follows in line with workflow hints in the docs contributor guideline intermediate contributing page, https://kubernetes.io/docs/contribute/intermediate/.
- nit: TL;DR, not sure if that phrase is meaningful here 😄

Reply:
We could scratch it and just leave it as 'It is a quick reference of useful...'

Reply:
Sure. This is a great pocket guide, totally useful.